### PR TITLE
Add CESM monthly time axis issue to FAQ

### PR DESCRIPTION
**Pull request by andersy005**

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -132,6 +132,64 @@ Distributed writes to netCDF are hard.
 1. If you need to write to netCDF and your final dataset can fit in memory then use `dataset.load().to_netcdf(...)`.
 1. If you really must write a big dataset to netCDF try using `save_mfdataset` (see [here](https://ncar.github.io/xdev/posts/writing-multiple-netcdf-files-in-parallel-with-xarray-and-dask/)).
 
+### Fixing CESM monthly data time axis
```
Comment (andersy005) [inline faq.md:135]: Couldn't come up with a better title :)... Let me know if this is good enough.

File: faq.md [hunk 2]
```diff
@@ -132,6 +132,64 @@ Distributed writes to netCDF are hard.
...
+1. instructing xarray not to decode the time via the `decode_times=False` option
+2. computing new time axis by averaging the time bounds
+3. manually decoding the newly computed time axis via `xr.decode_cf()`
+
+```python
+In [35]: ds = xr.open_dataset(filename, decode_times=False)
+
+In [36]:  attrs = ds.time.attrs.copy() # Make sure to keep our original attributes
+
+In [37]: ds = ds.assign_coords(time= ds.time_bnds.mean('nbnd'))
```
Comment (dcherian) [inline faq.md:171]: `'nbnd'` is hardcoded. Alternatively could use
https://github.com/xarray-contrib/cf-xarray/blob/e4fa084e557ddfbb8f3cbba3df109009f73e5361/cf_xarray/accessor.py#L1568-L1587

Mention that in a comment or condense that to like 2 lines and put it in the example? :P

File: faq.md [hunk 3]
```diff
@@ -132,6 +132,64 @@ Distributed writes to netCDF are hard.
 1. If you need to write to netCDF and your final dataset can fit in memory then use `dataset.load().to_netcdf(...)`.
 1. If you really must write a big dataset to netCDF try using `save_mfdataset` (see [here](https://ncar.github.io/xdev/posts/writing-multiple-netcdf-files-in-parallel-with-xarray-and-dask/)).
 
+### Fixing CESM monthly data time axis
+
+A well known issue of CESM data is that timestamps for fields saved as averages are placed at the end of the averaging period, and xarray does not treat this correctly when decoding the time axis. For instance, in the following example, the `January/1920` average has a timestamp of `February/1920`:
```
Comment (dcherian) [inline faq.md:137]: xarray is treating it correctly! The file says the time is `midnight Feb 1 1920` which is the non-inclusive end of the time bound...

Comment (andersy005) [reply faq.md:137]: > xarray is treating it correctly!  

You are right. I meant to mention that xarray isn't relying on the `bounds` information when decoding the `time` 

Removing this part shortly...

**Comment by mgrover1** [comment]
Would this be better to put in a separate ## CESM Data section?

File: faq.md (hunk 2 above)
Comment (andersy005) [reply faq.md]: Re-wrote everything with `cf-xarray` :)

**Review by mgrover1 — COMMENTED** [review]
Looks good!

**Pull request merged**